README.md

```diff
@@ -1,85 +1,118 @@
 # Poly Cookies
 
 ## A free solution to cookies consent
-[Polycookies.com](https://polycookies.com) is a free and open-source solution to cookies consent management. Implementing Privacy Protection on your website is not just for legal compliance, it's about caring for your users.
 
+[Polycookies.com](https://polycookies.com) is a free and open-source solution to
```

Comment:
Some changes were made in the documentation to align it better with the new files (like for example the renamed `cookies-consent.js` as `cookies-consent.html`), and also made some links more readable and added myself as an author tho I think it would be better to have a "collaborators" section and be there instead.